002_simulation_structure/002_simulation_structure.tex

```diff
@@ -1,14 +1,108 @@
+
+\begin{equation}\label{updateHP_general}
+   \text{newHP}= \text{currentHP} +\underbrace{w(1-e^{\frac{-\text{foodTaken}}{\tau}})}_{\text{HPChange}}
+\end{equation}
+
+where the two parameters $w$ and $\tau$ are defined at the beginning of the simulation. The shape of this curve is given in \Cref{fig:updateHP} together with some important parameters.
+
+\begin{figure}[htb]%
+    \centering
+    \subfloat[\centering Overview]{{\includegraphics[width=0.36\linewidth]{002_simulation_structure/images/health_updateHP_overview.pdf}}}%
+    \qquad
+    \subfloat[\centering Detailed representation]{{\includegraphics[width=0.36\linewidth]{002_simulation_structure/images/health_updateHP_detailed.pdf}}}%
+    \caption{\texttt{updateHP} as a function of the amount of food eaten (``FoodTaken'').}%
+    \label{fig:updateHP}%
+\end{figure}
+
+It is not possible to gain more HP than $w$ over the duration of one day; this is an intentional limit to prevent an agent's health from improving too quickly. As an example, we can think of an agent that starts from the weak level and wants to reach the maximum HP value. It would take several days for this agent to ``recover'' from this weak level and stabilise its health to a high HP value. 
+
+Note that it is possible for an agent to achieve an HP value that is larger than \texttt{MaxHP} inside \lstinline$hpDecay()$. At the end of each day, the \lstinline$hpDecay()$ function will apply the cost of living and then bound the final HP value by \lstinline$MaxHP$.
+
+
+Agents in the critical state are treated differently. For these agents, HP is updated according to this equation:
+
+\begin{equation}\label{updateHP_critical}
+    \text{newHP} = \min\left\{\text{HPCritical}+\text{HPReqCToW}, \text{currentHP} +w(1-e^{\frac{-\text{foodTaken}}{\tau}})\right\}
+\end{equation}
+
+\subsection{Cost of Living: \lstinline$hpDecay()$}\label{hpDecay}
+At the end of each day, the HP value of the agents will be reduced by the cost of living. The cost of living is larger for an agent with larger HP value than for an agent with lower HP value. This fact is motivated by a simple observation: humans that have stronger bodies and immune systems also need more food to sustain their level of health. The exact relation between HP value, cost of living, and HP value after applying the cost of living is given by the following linear relation:
+
+\begin{equation}\label{hpDecay_equation}
+    \text{newHP} = \text{currentHP}-\left[b + s(\text{currentHP}-\text{WeakLevel})\right]
+\end{equation}
+
+
+where $b$ is a (constant) base cost, and $s$ is the slope of the linear function. These parameters are initialised at the beginning of the simulation. \ToDo{Add a diagram for this?}
+
+To ensure that the HP value at the end of the day is bounded by \texttt{MaxHP}, we slightly modify the above expression:
+
```

Comment:
I think the following description could be explained with regular text and a state diagram of some sort. The equations don't really say much here.

Reply:
I think it is actually ok to have it for completeness. But I agree, this is not the most important equation

Reply:
yeah thats fair, if we keep then we should at least keep a consistent reference system to the equations, i.e. refer to equations in text using(eqn_number) instead of by position.